Aardvark/ARKEmailBugReporter.m

```diff
@@ -489,6 +489,7 @@ - (void)_dismissEmailComposeWindow;
     
     [self.mailComposeViewController.view removeFromSuperview];
     self.emailComposeWindow.rootViewController = nil;
+    self.emailComposeWindow.hidden = YES;
```

Comment:
Why does setting it to hidden here matter, given we nil it out on the next line?

Reply:
I think unless we do this, the window is retained by the system, so the = nil does nothing?

Reply:
(Eg, the act of showing a window registers it with UIKit, and hiding it unregisters is – so that decrements the retain count, so then the = nil will properly remove the window and dealloc it.)

Reply:
yeah, this seems like the mostly likely cause… I also tried moving the end of the appearance transition to after we change the key window, but that didn't help either

Reply:
Ah, makes sense. Maybe worth adding a comment in the code explaining that?

Reply:
done! 🙏